### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by lucascomamala**
- Added current_user method, which returns the first user from the database for the moment
- Created forms to perform the following functions:
  - Creates a Post on behalf of the current_user
  - Create a comment on behalf of the current_user
- Added functionality to allow Users to add likes to Posts.
- Added functionality to allow Users to remove previously added likes from Posts.

### CODE REVIEW

File: app/views/posts/_new_comment.html.erb
Comment (KDlamini) [inline app/views/posts/_new_comment.html.erb:1]: There's an error in creating new comments. Kindly debug the comments controller and make sure that it is set up correctly. The comments form should be able to submit inputs to the database successfully.

image

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  private
+
+  def find_post
+    @post = Post.find(params[:post_id])
+  end
+
+  def already_liked?
+    Like.where(user_id: @current_user.id, post_id: params[:post_id]).exists?
+  end
```
Comment (KDlamini) [inline app/controllers/likes_controller.rb:31]: 🚀 Great job on going above and beyond by adding likes validations to ensure that each user can only like a post once. 
👏🏾 💯 Kudos on adding the unlike functionality too 🥳

File: db/seeds.rb
Comment (KDlamini) [inline db/seeds.rb:1]: Well done on your effort to create adequate user data for populating the database. This makes testing your application a lot easier 🥂🎊 

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @lucascomamala 🙋‍♀️,  

Great work so far! 💯 
You have done a great job adding forms to the Rails Blog app. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞

✔️  Professional pull request and README
✔️  Forms in the views that accepts input for creating posts and comments implemented
✔️  Controller actions for creating new posts, comments, and likes present
✔️  Respective routes for the new actions added

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @lucascomamala , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

power approve


## Highlights
Nice design :+1: 
Great effort in making the requested changes :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**